### DISCUSSION TITLE
Why is the "trf" model more accurate in tems of prediction than the "lg" model while it is smaller in size than the latter?

### DISCUSSION THREAD

**Discussion by qingyun1988** [explosion/spaCy#8680] [Language Support]
Why is the "trf" model more accurate in terms of prediction than the "lg" model while it is smaller in size than the latter?

What is the difference between them?

And, Why is that the 'lg' model sometimes can be changed after training on it by using only a few(even one) examples while the 'trf' model can't be changed even using a lot of examples? After all, the 'trf' model is smaller than 'lg' one.

I wonder why this is. Thank you.

**Comment by svlandeg** [comment]
Hi,

The "trf" model is a transformer-based pipeline which has higher accuracy in general because transformers are generally more powerful than then the more simple "tok2vec"-based components in the sm/md/lg pipelines. You can find more details in the documentation.

With respect to your other question, please refer to [Paul's post](https://github.com/explosion/spaCy/discussions/8556) and our request to please stop asking the same question over and over again. To avoid any further duplicate conversations, I will be locking this conversation and I urge you again to please stop reposting this. We cannot continue supporting you over this forum in this way.
